schema/draft/schema.ts

```diff
@@ -267,6 +267,22 @@ export interface ServerCapabilities {
   };
 }
 
+interface Icons {
+  src: string;
+  /**
+   * The MIME type of the icon.
+   */
+  mimeType?: string;
+  /**
+   * A string that specifies one or more sizes at which the icon file can be used.
+   * Each size is specified as <width in pixels>x<height in pixels>. If multiple sizes are specified, they are separated by spaces; for example, 48x48 96x96.
+   * For vector formats like SVG, you can use any to indicate scalability
```

Comment:
i borrowed heavily from the [the web manifest docs here}(https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Manifest/Reference/icons) 😅